### PR TITLE
override pkg hash for both variants

### PR DESCRIPTION
**Pull request by 0xbe7a**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

I've uploaded both builds to https://anaconda.org/be7a/botocore/files and they have different hashes

### CODE REVIEW

**Comment by tkelman** [comment]
Thanks!

The naming/sense of the selector feels a little off to me in this, like technically the `python >=3.8` variant could get installed in a python 3.10 or newer environment if that environment doesn't require urllib3 v2

**Comment by 0xbe7a** [comment]
Closing in favor of #1485

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,24 +10,24 @@ source:
...
-    - python >=3.10  # [python_version == "new"]
+    - python >=3.8  # [python_pin_version == "geq38"]
+    - python >=3.10  # [python_pin_version == "geq10"]
     - jmespath >=0.7.1,<2.0.0
     - python-dateutil >=2.1,<3.0.0
-    - urllib3 >=1.25.4,<1.27  # [python_version == "old"]
+    - urllib3 >=1.25.4,<1.27  # [python_pin_version == "geq38"]
     # https://github.com/boto/botocore/pull/3034
-    - urllib3 >=1.25.4,<2.1  # [python_version == "new"]
+    - urllib3 >=1.25.4,<2.1  # [python_pin_version == "geq10"]
```
Comment (tkelman) [inline recipe/meta.yaml:30]:
```suggestion
    - urllib3 >=1.25.4,<2.1  # [python_pin_version == "geq310"]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,24 +10,24 @@ source:
...
-    - python >=3.8  # [python_version == "old"]
-    - python >=3.10  # [python_version == "new"]
+    - python >=3.8  # [python_pin_version == "geq38"]
+    - python >=3.10  # [python_pin_version == "geq10"]
     - pip
   run:
-    - python >=3.8  # [python_version == "old"]
-    - python >=3.10  # [python_version == "new"]
+    - python >=3.8  # [python_pin_version == "geq38"]
+    - python >=3.10  # [python_pin_version == "geq10"]
```
Comment (tkelman) [inline recipe/meta.yaml:25]:
```suggestion
    - python >=3.10  # [python_pin_version == "geq310"]
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -10,24 +10,24 @@ source:
...
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
-
+  string: py{{ python_pin_version }}{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
 requirements:
   host:
     # selectors break the linter but it's okay in this case
-    - python >=3.8  # [python_version == "old"]
-    - python >=3.10  # [python_version == "new"]
+    - python >=3.8  # [python_pin_version == "geq38"]
+    - python >=3.10  # [python_pin_version == "geq10"]
```
Comment (tkelman) [inline recipe/meta.yaml:21]:
```suggestion
    - python >=3.10  # [python_pin_version == "geq310"]
```

**Review by tkelman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**